Odata-docs/webapi-8/fundamentals/routing-overview.md

```diff
@@ -0,0 +1,365 @@
+---
+title:  "Routing in ASP.NET Core OData 8"
+description: Routing in ASP.NET Core OData 8.
+date:   2022-12-07
+ms.date: 12/7/2022
+author: gathogojr
+ms.author: jogathog
+---
+
+# Routing in ASP.NET Core OData 8
+**Applies To**:[!INCLUDE[appliesto-webapi](../../includes/appliesto-webapi-v8.md)]
+
+This tutorial covers routing in ASP.NET Core OData 8. This tutorial assumes a basic understanding of how to create and run an ASP.NET Core OData 8 application. If you're unfamiliar with ASP.NET Core OData 8, you may want to go through the [Getting started](/odata/webapi-8/getting-started) tutorial.
+
+## Prerequisites
+
+[!INCLUDE[](../../includes/appliesto-webapi-v8-net-prereqs-vs.md)]
```

Comment:
@gathogojr  do you think that we might want to point to the routing article for ASP.NET? I found it hard to follow along the OData concepts until after I read this https://learn.microsoft.com/en-us/aspnet/core/fundamentals/routing?view=aspnetcore-6.0